### PR TITLE
Copy of client xmpp project

### PR DESCRIPTION
**Pull request by secondsun**

### CODE REVIEW

**Comment by secondsun** [comment]
@danielpassos @danbev @matzew :eyes: ?

Some notes:
1) The project inherits from the aerogear-android parent instead of the previous sync parent.  
2) The project is in a subdirectory and not the root directory.  I don't see any reason to move it to root but I don't see any reason to pull it out.

**Comment by danbev** [comment]
:+1: Apart for the minor comments above I think we can merge this and take it from there.

File: client-xmpp/pom.xml [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+  See the License for the specific language governing permissions and
+  limitations under the License.
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    
+    <parent>
+        <groupId>org.jboss.aerogear</groupId>
+        <artifactId>aerogear-parent</artifactId>
+        <version>0.2.10</version>
```
Comment (matzew) [inline client-xmpp/pom.xml]: let's use 0.2.11 :-)

**Comment by matzew** [comment]
> The project inherits from the aerogear-android parent instead of the previous sync parent.

great!

**Comment by matzew** [comment]
Not sure we need this: https://github.com/secondsun/aerogear-android-sync/tree/master/javascript

Also, let's at least have a little README

**Comment by secondsun** [comment]
@matzew That is an old of mine and not the origin of this PR.

**Comment by matzew** [comment]
> @matzew That is an old of mine and not the origin of this PR.

ok, let's remove the JS folder - and perhaps move the code up to top level ?

**Comment by secondsun** [comment]
@matzew There is no JS folder in this PR.  You are looking at the wrong upstream repo.

**Comment by matzew** [comment]
LOL! :)

On Fri, Jan 23, 2015 at 8:27 AM, Hoyt Summers Pittman <
notifications@github.com> wrote:

> @matzew <https://github.com/matzew> There is no JS folder in this PR. You
> are looking at the wrong upstream repo.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/aerogear/aerogear-android-sync/pull/1#issuecomment-71156811>
> .
>



-- 
Matthias Wessendorf

blog: http://matthiaswessendorf.wordpress.com/
sessions: http://www.slideshare.net/mwessendorf
twitter: http://twitter.com/mwessendorf

**Pull request merged**